jax/_src/random.py

```diff
@@ -991,6 +1045,7 @@ def gamma(key: KeyArray,
       produces a result shape equal to ``a.shape``.
     dtype: optional, a float dtype for the returned values (default float64 if
       jax_enable_x64 is true, otherwise float32).
+    log_spasce: optional, if True, return the samples in log space.
```

Comment:
typo `log_spasce` :)
I think it is better to have [loggamma](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.loggamma.html) sampler rather than adding an argument here.

Reply:
Hmm, I thought loggamma generates log(gamma samples). Probable the name is confusing. For lognormal, we generate samples from exp(normal samples), but that pattern does not apply for loggamma. Probably statisticians should rename LogNormal to ExpNormal for a better interpretation. I searched the question and found this [stackexchange](https://math.stackexchange.com/questions/4268139/why-is-it-called-a-lognormal-distribution-instead-of-an-expnormal-distribution) thread.

Reply:
It looks like (according to scipy's definition) log-gamma gives just what we need: https://github.com/scipy/scipy/blob/0ed66c76c010ec897a116e5a331941e43e3da562/scipy/stats/_continuous_distns.py#L4917

Reply:
Still I'm not sure it would make sense to create a new `random_loggamma_p` primitive since the implementation would share so much with the existing `gamma_p` primitive. What if we add the keyword to the primitive, but expose it to the user via a `jax.random.loggamma()` which has the property `jnp.allclose(jnp.exp(loggamma(*args)), gamma(*args))`?

Reply:
> add the keyword to the gamma primitive, but expose it to the user via a jax.random.loggamma()

Agreed! I think we don't need to expose that argument to the user api. I just suggested removing `log_space` from this `gamma` function, not the primitive.

Reply:
Done